compiler/src/iree/compiler/Dialect/LinalgExt/IR/test/roundtrip.mlir

```diff
@@ -592,6 +592,56 @@ func.func @scatter_update_slice_2D(
 
 // -----
 
+func.func @gather_static(
+    %source : tensor<10xf32>, %idx : tensor<1xi32>,
+    %result : tensor<1xf32>) -> tensor<1xf32> {
+  %0 = iree_linalg_ext.gather
+    dimension_map = [0]
+    ins(%source, %idx : tensor<10xf32>, tensor<1xi32>)
+    outs(%result : tensor<1xf32>) {
+    ^bb0(%arg0: f32, %arg1: f32):
+      iree_linalg_ext.yield %arg0 : f32
+  } -> tensor<1xf32>
+  return %0 : tensor<1xf32>
+}
+// CHECK-LABEL: func.func @gather_static(
+// CHECK-SAME:   %[[SOURCE:[a-zA-Z0-9_]+]]
+// CHECK-SAME:   %[[IDX:[a-zA-Z0-9_]+]]
+// CHECK-SAME:   %[[RESULT:[a-zA-Z0-9_]+]]
+//     CHECK:   %[[VAL:.+]] = iree_linalg_ext.gather
+// CHECK-SAME:     dimension_map = [0]
+// CHECK-SAME:     ins(%[[SOURCE]], %[[IDX]]
+// CHECK-SAME:     outs(%[[RESULT]]
+//     CHECK:     iree_linalg_ext.yield %{{.+}} : f32
+//     CHECK:   return %[[VAL]]
```

Comment:
I think you aim to align the checks with `:`, and there are missing indents? (Same for the below test case)